### PR TITLE
Extra parameter for UpdateImageStore

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR partly ...

Fixes: #8391
By allowing operators to update the capacity of image stores

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)
- [ ] build/CI

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test alma9 kvm-alma9

**Review by rohityadavcloud — DISMISSED** [review]
Didn't test but LGTM, some remarks:
- Can you also add ability to change the name of the image (secondary) storage?
- Add support of a simple edit form in the UI

**Comment by DaanHoogland** [comment]
> Didn't test but LGTM, some remarks:
> 
>     * Can you also add ability to change the name of the image (secondary) storage?
> 
>     * Add support of a simple edit form in the UI

sure :D , work underway you don't have to :-1:  for that ;)

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan package

File: ui/src/views/AutogenView.vue [hunk 1]
```diff
@@ -679,7 +679,6 @@ export default {
     },
     getOkProps () {
       if (this.selectedRowKeys.length > 0 && this.currentAction?.groupAction) {
-        return { props: { type: 'default' } }
```
Comment (vishesh92) [inline ui/src/views/AutogenView.vue:682]: Why do we need to remove this?

Comment (DaanHoogland) [reply ui/src/views/AutogenView.vue:682]: it is now in the edit dialog (we don't need to)

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by vishesh92 — APPROVED** [review]
clgtm. didn't test.

**Comment by DaanHoogland** [comment]
@andrijapanicsb , can you test this to see if this satisfies your request in #8391 ?

**Comment by kiranchavala** [comment]
@DaanHoogland can you point the PR to 4.19

**Comment by DaanHoogland** [comment]
> @DaanHoogland can you point the PR to 4.19

I could but it applies to 4.18 as well.

**Comment by kiranchavala** [comment]
@DaanHoogland 

As discussed, the issue  #8391  needs more functional description.

During testing of the PR, I found that we are not updating the "op_host_capacity"  table  of capacity type 6

https://cloudstack.apache.org/api/apidocs-4.19/apis/listCapacity.html

Also the API call "update imagestore"  doesn't update the disksizetotal and disksizeused

```
update imagestore capacitybytes=4395899027457 id=01357148-64a7-49f6-8f60-9f04718309ed
{
  "imagestore": {
    "disksizetotal": 2197949513728,
    "disksizeused": 1300025573376,
    "hasannotations": false,
    "id": "01357148-64a7-49f6-8f60-9f04718309ed",
    "name": "nfs://10.0.32.4/acs/secondary/ref-trl-6809-k-Mr8-kiran-chavala/ref-trl-6809-k-Mr8-kiran-chavala-sec1",
    "protocol": "nfs",
    "providername": "NFS",
    "readonly": false,
    "scope": "ZONE",
    "url": "nfs://10.0.32.4/acs/secondary/ref-trl-6809-k-Mr8-kiran-chavala/ref-trl-6809-k-Mr8-kiran-chavala-sec1",
    "zoneid": "76afe9f4-8860-4e0d-bf68-d973637f59c2",
    "zonename": "ref-trl-6809-k-Mr8-kiran-chavala"
  }
}
```

cc @andrijapanicsb

**Review by rohityadavcloud — APPROVED** [review]
LGTM - this is same as for primary/storage pools

**Pull request merged**